gammapy/maps/geom.py

```diff
@@ -417,6 +425,11 @@ def unit(self):
         """Return coordinate axis unit."""
         return self._unit
 
+    @property
+    def type(self):
```

Comment:
Please add a test for this type, or ideally even one that covers the three cases of "energy", "time" and "any".